README.md

```diff
@@ -4,7 +4,7 @@ Final Project for AMATH 495
 
 The project is based on the paper [DiffMIC: Dual-Guidance Diffusion Network for Medical Image Classification](https://arxiv.org/abs/2303.10610).
 
-`main.py` calls the modules in `src/`. The program loads a saved DCG checkpoint `saved_dcg.pth` if it exists. It trains and saves a DCG checkpoint otherwise. Similarly, for diffusion, the program loads a saved diffusion checkpoint `saved_diff.pth` if it exists. It trains and saves a diffusion checkpoint otherwise. After loading the DCG and diffusion models, the program runs inference on the test images and outputs the predicted classification [Classes 1 - 5].
+`main.py` calls the modules in `src/`. The program loads a saved DCG checkpoint `saved_dcg.pth` if it exists. It trains and saves a DCG checkpoint otherwise. Similarly, for diffusion, the program loads a saved diffusion checkpoint `saved_diff.pth` if it exists. It trains and saves a diffusion checkpoint otherwise. After loading the DCG and diffusion models, the program runs inference on the test images and outputs the predicted classification [Classes 0 - 4].
```

Comment:
There's 5 classes.

Reply:
But the class number start from 0 to 4. Namely [0,1,2,3,4]

Reply:
That's okay. I chose 1-5 to show that it has 5 classes.

Reply:
It might be a bit misleading since classes are 0, 1, 2, 3, 4. We can mention [5 Classes: 0, 1, 2, 3, 4] to denote there are 5 classes.

Reply:
Okay, that works.

Reply:
Made the change.